regression/cbmc/gcc_builtin_sub_overflow/test.desc

```diff
@@ -0,0 +1,44 @@
+CORE gcc-only
+main.c
+
+\[check_int.assertion.1\] line \d+ assertion !__builtin_ssub_overflow\(1, 1, &result\): SUCCESS
+\[check_int.assertion.2\] line \d+ assertion result == 0: SUCCESS
+\[check_int.assertion.3\] line \d+ assertion __builtin_ssub_overflow\(.*, 1, &result\): SUCCESS
```

Comment:
In case you’re wondering, the regexes here are due cbmc on travis (and apparently _only_ on travis) “helpfully” expanding macros in assertion messages...